.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
   # The official reuse hook only supports calling lint, so we need our own hook
   - id: reuse-annotate
     name: reuse-annotate
-    entry: reuse annotate --license BSD-3-Clause --copyright 'The Ginkgo authors' --year 2017-2023 --style c
+    entry: reuse annotate --license BSD-3-Clause --copyright 'The Ginkgo authors' --year 2024 --style c --merge-copyright
```

Comment:
While we're at it, can we automate the year parameter in a platform portable way? e.g., in python something like
```
from datetime import datetime
datetime.now().strftime('%Y')
```
(maybe with timezone UTC)

Reply:
I think this would be something that needed to be changed in reuse. I will see, if I can get a PR there.